Index.html

```diff
@@ -0,0 +1,97 @@
+                                        mother, Vernita Lee (1935–2018), was a housemaid</span> </li>
+                                <li><span> Winfrey's biological father is usually noted as Vernon Winfrey (born c.
+                                        1933)</span></li>
+                                <li><span>1925</span>-Noah Robinson Sr. has claimed to be her biological father</li>
+                                <li><span>1986</span>-Winfrey has stated she was molested by her cousin, uncle, and a family
+                                    friend, starting when she was nine years old, something she first announced on episode
+                                    of her TV show regarding sexual abuse</li>
+                                <li>1990<span>-Winfrey later stated she felt betrayed by the family member who had sold the
+                                        story of her son to the National Enquirer.</span></li>
+                                <li><span> She attended Lincoln High School in Milwaukee, but after early success in the
+                                        Upward Bound program, was transferred to the affluent suburban Nicolet High
+                                        School</span></li>
+                                <li><span> Her first job as a teenager was working at a local grocery store. </span></li>
+                                <li><span>At the age of 17, Winfrey won the Miss Black Tennessee beauty pageant</span></li>
+                                <li><span>She also attracted the attention of the local black radio station, WVOL, which
+                                        hired her to do the news part-time.She worked there during her senior year of high
+                                        school and in her first two years of college.</span>
+                                </li>
+                                <li><span>
+                                    Winfrey's career in media would not have surprised her grandmother, who once said that ever
+                                since Winfrey could talk, she was on stage. As a child, she played games interviewing her
+                                corncob doll and the crows on the fence of her family's property. Winfrey later acknowledged
+                                her grandmother's influence, saying it was Hattie Mae who had encouraged her to speak in
+                                public and "gave me a positive sense of myself
+                                </span> </li>
+                            </ul>    
+                        </div>
+                    </section>
+            </main>
+        </div>
+        
+
+        <footer>
+            <section class="info">
+                <p>You can read more about Oprah Winfrey <a href="https://en.wikipedia.org/wiki/Oprah_Winfrey">here</a>.
+                </p>
+            </section>
+        </footer>
+
```

Comment:
![attrinute page](Your page)
![Uploading content.png…](The temple page )

This doesn't seem to be the same as in template